app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+
+ */
+@import "foundation_and_overrides";
+
+$background-color: #000022;
+$font-color: #ddddcc;
+$visited-color: #cc4411;
+$hover-color: #999999;
+$unvisited-color: #EC9C2C;
+
+body {
+  background: $background-color;
+  color: $font-color;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  color: $font-color;
+  text-align: center;
+}
+
+svg text {
+  font-size: 65%;
+  color: $font-color;
+}
+
+a:link {
+  color: $unvisited-color;
+}
+
+a:visited {
+  color: $visited-color;
+}
+
+a:hover {
+  color: $hover-color;
+}
+
+.field label {
+  color: $font-color;
+  font-size: 1em;
+}
+
+.field input {
+    border-radius: 3px;
+}
+
+.actions input {
+  background: $visited-color;
+  color: $font-color;
```

Comment:
Properties should be ordered background, border, border-radius, color, cursor, font, font-size